tests/staticfiles_tests/test_checks.py

```diff
@@ -132,3 +136,57 @@ def test_nonexistent_directories(self):
             # Nonexistent directories are skipped.
             finder = get_finder("django.contrib.staticfiles.finders.FileSystemFinder")
             self.assertEqual(list(finder.list(None)), [])
+
+
+class StoragesCheckTests(SimpleTestCase):
+    @override_settings(STORAGES={})
+    def test_error_empty_storages(self):
+        # "default" and "staticfiles" need to be popped from STORAGES since
+        # UserSettingsHolder has code to maintain backward compatibility.
+        # RemovedInDjango51Warning
+        settings.STORAGES.clear()
+        assert settings.STORAGES == {}
+        errors = check_storages(None)
+        self.assertEqual(
+            errors, [Error(MISSING_STORAGE_SETTING, id="staticfiles.E005")]
+        )
+
+    def test_no_errors_storages_default_value(self):
+        assert sorted(settings.STORAGES.keys()) == ["default", "staticfiles"]
+        errors = check_storages(None)
+        self.assertEqual(errors, [])
```

Comment:
I don't see much value in this test.

Reply:
It's ensuring that the default value for `STORAGES` does not trigger the system check.

Reply:
We normally don't test default settings without providing `@override_settings` as anyone can change it in their test settings, so we should add:
```python
@override_settings(STORAGES = {
    "default": {
        "BACKEND": "django.core.files.storage.FileSystemStorage",
    },
    "staticfiles": {
        "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
    },
})
```
However, for me, this is redundant with `test_no_errors_staticfiles_only`. Also, I'd rename `test_no_errors_staticfiles_only` to `test_staticfiles_no_errors` and it's not importanc, and doesn't give us any extra test coverage, that we only have `staticfiles` defined.

Reply:
Ok, removing `test_no_errors_storages_default_value` and renaming the other one. All pushed.